### PR TITLE
fix links in the Footer

### LINKED ISSUE #75: Old Gitlab Links in the Footer
"Repository" and "Report an issue" links in the footer are linked to old Gitlab links instead of the Github links.

### PR DESCRIPTION
**Pull request by ayushsatyam146**
Fixes #75 . Updated "Repository" and "Report an issue" Links in the Footer from GitLab to Github links.

### CODE REVIEW

File: src/utils/Links.store.js [hunk 1]
```diff
@@ -20,15 +20,15 @@ export default [
       },
       {
         name: 'Repository',
-        href: 'https://gitlab.com/SUSE-UIUX/eos-icons',
+        href: 'https://github.com/EOS-uiux-Solutions/eos-icons-landing',
         category: 'External link',
         action: 'Link to Gitlab repo',
```
Comment (kbhutani0001) [inline src/utils/Links.store.js:25]:
```suggestion
        action: 'Link to Github repo',
```

File: src/utils/Links.store.js [hunk 2]
```diff
@@ -20,15 +20,15 @@ export default [
...
         label: 'Footer',
         target: '_blank'
       },
       {
         name: 'Report an issue',
-        href: 'https://gitlab.com/SUSE-UIUX/eos-icons/issues',
+        href: 'https://github.com/EOS-uiux-Solutions/eos-icons-landing/issues',
         category: 'External link',
         action: 'Link to Gitlab issues',
```
Comment (kbhutani0001) [inline src/utils/Links.store.js:33]:
```suggestion
        action: 'Link to Github issues',
```

Comment (ayushsatyam146) [reply src/utils/Links.store.js]: Fixed this too.

File: src/utils/Links.store.js (hunk 1 above)
Comment (ayushsatyam146) [reply src/utils/Links.store.js]: Fixed this.

File: src/utils/Links.store.js (hunk 2 above)
Comment (rheo-chiti) [reply src/utils/Links.store.js]: LGTM merging

**Pull request merged**